src/plugins/provider.js

```diff
@@ -42,11 +42,16 @@ function createVueHooksMap (hookCallback) {
 
 function ngVueProvider ($injector) {
   let inQuirkMode = false
+  let vuexStore = null
```

Comment:
I hope the quirk mode is a plugin, the vuex is also a plugin, and any other features added to ngvue are all plugins.